Docs/source/Overview/software-overview.rst

```diff
@@ -0,0 +1,37 @@
+*****************
+Software Overview
```

Comment:
The BPS docs has a section about RTOS, should that also be included in here?

Reply:
Included RTOS